docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -7,7 +7,19 @@ ms.date: 09/20/2017
 
 # The history of C# #
 
-What did the language look like in its earliest incarnations? And how has it evolved in the years since?
+This article provides a history of each major release of the C# language. The C# team is continuing to innovate and add new features. Detailed language feature status, including features considered for upcoming releases can be found [on the dotnet/roslyn repository](https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md) on GitHub.
+
+> [!IMPORTANT]
+> The C# language relies on types and methods in a *standard library* for some of the features. One example is exception processing. Every `throw` statement or expression is checked to ensure the object being thrown is derived from <xref:System.Exception>. Similarly, every `catch` is checked to ensure that the type being caught is derived from <xref:System.Exception>. Each version may add new requirements. To use the latest language features in older environments, you may need to install specific libraries. These dependencies are documented in the page for each specific version. You can learn more about the [relationships between language and library](relationships-between-language-and-library.md) for background on this dependency.
```

Comment:
a standard library, or standard libraries?

Reply:
This is an interesting problem.  The C# Spec refers to a "standard library" for the types that are required. In practice, those types are delivered in multiple binary files, or "libraries". That's why this paragraph has the number mismatch: In the first sentence, I'm referring to the library from the standard. In the later sentence, I'm referring to the delivery method.

What do you think is the best resolution?

Reply:
The basic problem here, I think, is that naming conventions in the C# spec sometimes diverge from naming conventions for the .NET platform. I think I might say "types and method in what the C# specification defines as a \*standard library\*", with a link to the relevant portion of the spec.